pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Schema.java

```diff
@@ -167,49 +167,95 @@ default T decode(byte[] bytes, byte[] schemaVersion) {
     }
 
     /**
-     * Create a Avro schema type by extracting the fields of the specified class.
+     * Create a allow null Avro schema type by extracting the fields of the specified class.
```

Comment:
> Create a allow null Avro schema  

The phrase doesn't sound correct. Also, we should explain here what does that mean to a user since "allowNull" as an internal implementation detail of Avro, for which users might not be familiar with